src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/RejectVisitRequestBodyDto.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotNull
+
+data class RejectVisitRequestBodyDto(
+  @field:Schema(description = "Reference of the visit for rejection", required = true)
+  @field:NotNull
+  val visitReference: String,
+  @field:Schema(description = "Username for user who actioned this request", required = true)
+  @field:NotNull
+  val actionedBy: String,
+)
```

Comment:
can we rename and reuse ApproveVisitRequestBodyDto? Can leave is as 2 distinct if we feel more parameters might be added to either request body.